controllers/defaults/defaults.go

```diff
@@ -24,6 +24,15 @@ const (
 	// with a value of "false" to disable key rotation. When present, this annotation is then
 	// propagated to the associated StorageClasses.
 	KeyRotationEnableAnnotation = "keyrotation.csiaddons.openshift.io/enable"
+	// Identifies storageconsumer was upgraded from internal or provider mode or none for no compatilbity (new)
+	StorageConsumerBackwardCompatibleAnnotation   = "ocs.openshift.io/storageconsumer-4.18-backward-compatible-mode"
+	StorageConsumerBackwardCompatibleModeInternal = "internal"
+	StorageConsumerBackwardCompatibleModeProvider = "provider"
+	StorageConsumerBackwardCompatibleModeNone     = "none"
```

Comment:
This two are not used, Do we need it?

Reply:
`none` is being used and `provider` is the base for upcoming PRs and ya that single line can be removed.